hyper/http11/connection.py

```diff
@@ -137,6 +144,33 @@ def connect(self):
 
         return
 
+    @classmethod
```

Comment:
So this is weird, I know, but I kind of hate classmethods for this use case. IMO, classmethods are for custom constructors, and nothing else. This should just be a function that uses `HTTP11Connection` directly, I think.